bindings/python/tests/feature_model/test_feature_model.py

```diff
@@ -90,3 +91,96 @@ def test_iter_children(self):
         self.assertTrue(test_feature_root.is_child(test_feature_a))
         self.assertTrue(test_feature_root.is_child(test_feature_b))
         self.assertTrue(test_feature_root.is_child(test_feature_c))
+
+    def test_merge_models_idempotence(self):
+        """ Checks if a feature model merged with itself is unchanged. """
+        test_merged = self.fm.merge_with(self.fm)
+
+        self.assertTrue(test_merged)
+        self.assertTrue(test_merged.get_root())
+        self.assertTrue(test_merged.get_feature("A"))
+        self.assertTrue(test_merged.get_feature("AA"))
+        self.assertTrue(test_merged.get_feature("AB"))
+        self.assertTrue(test_merged.get_feature("AC"))
+        self.assertTrue(test_merged.get_feature("B"))
+        self.assertTrue(test_merged.get_feature("C"))
+        self.assertTrue(test_merged.get_feature("N"))
+
+    def test_merge_models(self):
+        """ Checks if merging a feature model with another works correctly. """
+        with open(TEST_INPUTS_DIR / "simple_example_feature_model_merge.xml",
+                  'r') as fm_file:
+            parser = fm_parsers.FeatureModelXmlParser(fm_file.read())
+            test_merge_fm = parser.build_feature_model()
+        test_merged = self.fm.merge_with(test_merge_fm)
+
```

Comment:
Can you make it clear what should change and what should not change. A small comment a separator or maybe adapt the function comment so we know what should happen here.